projects/rocprofiler-sdk/source/bin/CMakeLists.txt

```
@@ -26,6 +26,16 @@ install(
                 WORLD_EXECUTE
     COMPONENT tools)
 
+configure_file(rocprofv3_attach.py
```

```suggestion
configure_file(rocprofv3-attach.py
```
We use hyphens for executables, not underscores. Hyphens are far more common than underscores.